src/main/java/calculator/RunCalculator.java

```diff
@@ -0,0 +1,147 @@
+            throw new IllegalArgumentException("연산자 외 다른 값");
+        }
+    }
+
+    /**
+     * 사칙연산
+     * @param Operator
+     * @param a
+     * @param b
+     * @return
+     */
+    public static int commonCalculate(final char operator, final int first, final int second ){
+        int returnValue = 0;
+
+        switch(operator) {
+            case '+':
+                returnValue = CommonCalculate.addCalculate(first, second);
+                break;
+            case '-':
+                returnValue = CommonCalculate.subCalculate(first, second);
+                break;
+            case '*':
+                returnValue = CommonCalculate.mulCalculate(first, second);
+                break;
+            case '/':
+                returnValue = CommonCalculate.divCalculate(first, second);
+                break;
+        }
+
+        return returnValue;
+    }
+
+
+
+    /**
+     * 단계를 나누어 계산
+     */
+    public static int stepToCalculatorRun(final String paramText, final List<OperatorOrder> list ){
+        int returnValue = 0;
+
+        for(int i=0; i<list.size(); i++){
+            if(i == 0 ){
+                if(list.size() > 1){
+                    returnValue = RunCalculator.commonCalculate(list.get(i).getOperatorText()
+                            , Integer.valueOf(paramText.substring(0, list.get(i).getIndex()))
+                            , Integer.valueOf(paramText.substring(list.get(i).getIndex()+1, list.get(i+1).getIndex()))
+                    );
+                }else{
+                    returnValue = RunCalculator.commonCalculate(list.get(i).getOperatorText()
+                            , Integer.valueOf(paramText.substring(0, list.get(i).getIndex()))
+                            , Integer.valueOf(paramText.substring(list.get(i).getIndex()+1, paramText.length()))
+                    );
+                }
+            } else if(i < list.size() - 1){
+                returnValue = RunCalculator.commonCalculate(list.get(i).getOperatorText()
+                        , returnValue
+                        , Integer.valueOf(paramText.substring(list.get(i).getIndex()+1, list.get(i+1).getIndex()))
+                );
+            }else {
+                returnValue = RunCalculator.commonCalculate(list.get(i).getOperatorText()
+                        , returnValue
+                        , Integer.valueOf(paramText.substring(list.get(i).getIndex()+1, paramText.length()))
+                );
+            }
+        }
+
+        return returnValue;
+    }
+
+    /**
+     * 문자열을 받아서 연산자의 index와 연산자를 return
+     * @param param 문자열
+     * @return list - 연잔자 및 index의 list
+     */
+    public static List<OperatorOrder>  operatorDivision (final String param){
+        List<OperatorOrder> list = new ArrayList<OperatorOrder>();
+        OperatorOrder operOrder = new OperatorOrder();
+        char temp;
+
+        for(int i=0; i < param.toCharArray().length; i++){
```

Comment:
단항 연산자(++ 혹은 --)의 경우와 .을 제외한 모든 이항(binary) 연산자는 연산수들과는 공백으로 분리되어져야 합니다 :)

```suggestion
        for (int i = 0; i < param.toCharArray().length; i++) {
```